Automattic/jetpack
# My Jetpack: Move manage connection link

Fixes https://github.com/Automattic/jetpack-marketing/issues/911

## Proposed changes:

* move the `Manage` link next to the connection icons to avoid duplication and confusion next to the site and user connection

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a JN site pointing to this branch
* Connect Jetpack and your user
* Go to "My Jetpack"
* Verify that the "Manage" link is visible next to the connection icons with your avatar


<img width="535" alt="SCR-20240905-jryi" src="https://github.com/user-attachments/assets/f5bf7d75-a4b1-43c1-bba3-52065b7a7fb5">
